tests/build/build.go

```diff
@@ -493,7 +493,7 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 
 			It("should validate HACBS taskrun results", func() {
 				// List Of Taskruns Expected to Get Taskrun Results
-				gatherResult := []string{"conftest-clair", "sanity-inspect-image", "sanity-label-check"}
+				gatherResult := []string{"conftest-clair", "sanity-inspect-image", "sanity-label-check", "sbom-json-check"}
```

Comment:
conftest-clair task doesn't exist anymore. it was [renamed to clair-scan](https://github.com/redhat-appstudio/build-definitions/commit/c483db9d2bfd86daf7eeb9288bd6abf869bd5a7a#diff-0240a91e2ace74bf029b4153dcb9ba968842bb5a69706780a9d717d7ca8533b3L10)

Reply:
what does the clair-scan logs shows?

Reply:
"script": "if [ ! -s /tekton/home/clair-result.json ]; then\n  echo \"Previous step [get-vulnerabilities] failed, /tekton/home/clair-result.json is empty.\"\nelse\n  /usr/bin/conftest test --no-fail /tekton/home/clair-result.json \\\n  --policy /project/clair/vulnerabilities-check.rego --namespace required_checks \\\n  --output=json | tee /tekton/home/clair-vulnerabilities.json || true\nfi\n",

Reply:
clair-scan task itself is failing here, @MartinBasti @jsztuka could you guys take a look at the [logs](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/redhat-appstudio_e2e-tests/320/pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e/1621099087958904832/artifacts/redhat-appstudio-e2e/redhat-appstudio-gather/artifacts/taskruns.json ) here ?

Reply:
Not sure why the basic tekton folder structure is not present during time of the test, /tekton/home should be present at that time.

Reply:
And where exactly is it failing? I see only success status in clair-scan task and its steps
```
                            "exitCode": 0,
                            "finishedAt": "2023-02-02T12:00:36Z",
                            "message": "[{\"key\":\"HACBS_TEST_OUTPUT\",\"value\":\"{\\\"result\\\":\\\"SUCCESS\\\",\\\"timestamp\\\":\\\"1675339236\\\",\\\"note\\\":\\\"For more details please visit the logs in workspace of Tekton tasks.\\\",\\\"namespace\\\":\\\"required_checks\\\",\\\"successes\\\":4,\\\"failures\\\":0,\\\"warnings\\\":0}\\n\",\"type\":1}]",
                            "reason": "Completed",
```

Reply:
This seems to be addressed in the latest changes and working correctly.